torchtext/datasets/sogounews.py

```diff
@@ -36,9 +36,14 @@
 @_add_docstring_header(num_lines=NUM_LINES, num_classes=5)
 @_create_dataset_directory(dataset_name=DATASET_NAME)
 @_wrap_split_argument(('train', 'test'))
-def SogouNews(root, split):
+def SogouNews(root, split, offset=None):
     path = _download_extract_validate(root, URL, MD5, os.path.join(root, _PATH), os.path.join(root, _EXTRACTED_FILES[split]),
                                       _EXTRACTED_FILES_MD5[split], hash_type="md5")
     logging.info('Creating {} data'.format(split))
+    offset_split = 0
```

Comment:
This logic should be hidden in the _wrap_split_argument decorator

Reply:
Right, make sense. Completely missed the decorator. Thanks!